examples/crm/src/Layout.tsx

```diff
@@ -14,7 +14,7 @@ const Layout = ({ children }: LayoutProps) => (
             <main id="main-content">
                 {/* @ts-ignore */}
                 <ErrorBoundary FallbackComponent={Error}>
-                    {children}
+                    <Suspense fallback={<Loading />}>{children}</Suspense>
```

Comment:
I wonder if we shouldn't do that by default on all our layouts.

Reply:
Probably